app/models/count.rb

```diff
@@ -0,0 +1,7 @@
+# this table is just used for counting the rows
+# e.g.
+# name: ckb_transactions
+# value: 10014390
+# see: db/migrate/20221227013538_create_counts.rb
+class Count < ApplicationRecord
```

Comment:
This name is too common and may cause name conflict problem.
We use use a more specific name

Reply:
count_cache?